trade-finance-manager-ui/server/controllers/helpers/isFacilityEndDateEnabledForFacility.test.js

```diff
@@ -0,0 +1,45 @@
+import { isTfmFacilityEndDateFeatureFlagEnabled } from '@ukef/dtfs2-common';
+import { isFacilityEndDateEnabledForFacility } from './isFacilityEndDateEnabledForFacility';
+
+jest.mock('@ukef/dtfs2-common', () => ({
+  ...jest.requireActual('@ukef/dtfs2-common'),
+  isTfmFacilityEndDateFeatureFlagEnabled: jest.fn(),
+}));
+
+describe('isFacilityEndDateEnabledForTfmFacility', () => {
+  const gefFacility = {
+    facilitySnapshot: { type: 'Cash facility' },
+  };
+
+  const bssEwcsFacility = {
+    facilitySnapshot: { type: 'Loan' },
```

Comment:
can this and cash above come from a constant

Reply:
The Bond/Loan values can be taken from `FACILITY_TYPE` in libs/common but think you'll need to make a new one for the weird `'Cash facility'`

Reply:
Have changed to come from a constant